source/_integrations/opengarage.markdown

```diff
@@ -3,6 +3,7 @@ title: OpenGarage
 description: Instructions on how to integrate OpenGarage.io covers within Home Assistant.
 ha_category:
   - DIY
+ha_iot_class: Local Push
```

Comment:
This integration uses the default cover polling logic to trigger the `async_update` method of the entity, using a scan-interval (the default).

```suggestion
ha_iot_class: Local Push
```

Reply:
Looked to deep in the code, you know the architecture of covers ;D Will change!